src/xopen/__init__.py

```diff
@@ -732,6 +735,10 @@ def xopen(
     (parallel gzip) subprocess if possible. See PipedGzipWriter and PipedGzipReader.
 
     When threads = 0, no subprocess is used.
+
+    format overrides the autodetection of input and output formats. This can be
+    useful when compressed output needs to be written to a file without an
+    extension. Possible values are "gz", "xz" and "bz2".
```

Comment:
Is there a use case for forcing uncompressed input/output?

Reply:
No. You just use `open` from builtins in that case. No need to bother with `xopen`.